tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 downloadcache = {toxworkdir}/cache/
-envlist = py2.7-django1.5,py2.6-django1.5,py2.7-django1.4,py2.6-django1.4,py2.7-django1.3,py2.6-django1.3
+envlist = py3.4-django1.9, py3.4-django1.8, py2.7-django1.9,py2.7-django1.8,py2.7-django1.7,py2.7-django1.6,py2.7-django1.5,py2.6-django1.5,py2.7-django1.4,py2.6-django1.4,py2.7-django1.3,py2.6-django1.3
```

Comment:
Indeed, I am more used to this format in my own packages. I was trying to submit a minimal diff. I'll use the modern approach.